### ISSUE TITLE
[BUG] hvac_modes shown as heat_cool shown instead of heat.

### ISSUE THREAD

**Issue by bouwew** [plugwise/plugwise-beta#73]
**Describe the bug**
The climate-entities show hvac_modes: heat_cool, auto. Used to be heat, auto.

**Evidence**
![image](https://user-images.githubusercontent.com/11290930/83177200-ed447980-a11e-11ea-9bf5-8551b0c96e35.png)

``` 
**Expected behavior**
Show: heat instead of heat_cool.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Home Assistant (please complete the following information):**
 - Version: 0.110.3
 - Deployment: hassio
 - Installed plugwise version: 0.2.12rc1, not sure which version introduced this mistake.
 - Installed through HACS

**Smile (please complete the following information):**
 - Sort of Smile: [e.g. Adam/Anna/P1]
 - Firmware: [check through the app or the integrations page in Home Assistant]

**Additional context**
Add any other context about the problem here.

**Comment by bouwew** [comment]
This is the output from Smile:
```
2743216f626f43948deec1f7ab3b3d70 {'name': 'Adam', 'types': {'thermostat', 'temperature', 'home'}, 'class': 'heater_central', 'location': '07d618f0bb80412687f065b8698ce3e7'} {'heating_state': False, 'dhw_state': False, 'water_temperature': 40.0, 'intended_boiler_temperature': 0.0, 'cooling_state': False, 'flame_state': False}
```

Showing `climate = False`, this should not be shown --> bug in Smile.

**Comment by bouwew** [comment]
Update, my XML-data shows!
```
<point_log id="1ef800514a384f04a3317da47736be43">
<updated_date>2020-05-27T05:17:40.014+02:00</updated_date>
<type>cooling_state</type>
<unit/>
<last_consecutive_log_date>2020-05-27T05:17:40.014+02:00</last_consecutive_log_date>
<period start_date="2020-05-27T05:17:40.014+02:00" end_date="2020-05-27T05:17:40.014+02:00">
<measurement log_date="2020-05-27T05:17:40.014+02:00">off</measurement>
</period>
<boiler_state id="dd7213afa98a4d00bcc7f5e8a46dbdee"/>
```
So not a bug, but an addition to the Adam firmware, I'm on v3.0.23.

**Comment by bouwew** [comment]
Solved in #72, using compressor_state instead of cooling_state for detection of cooling-capability.

**Comment by bouwew** [comment]
Implemented in v0.2.13
